web/src/Pages/ProgrammeView/programmeView.tsx

```diff
@@ -1286,6 +1286,11 @@ const ProgrammeView = () => {
     calculations.constantVersion = data.constantVersion;
   }
 
+  const getFileName = (filepath: string) => {
+    const fileName = filepath.substring(0, filepath.lastIndexOf('/'));
```

Comment:
This will work for url https://www.w3schools.com/tags/default.pdf/.
Will this work https://www.w3schools.com/tags/default.pdf

Reply:
fixed